### PR TITLE
SEO-3569 Angular Controls Site Audit

### PR DESCRIPTION
**Pull request by Yvone-Atieno**
Hi @Aishwarya-Ganesan ,

Have changed title, description, H1 and named images.

Regards,
Yvone.

### CODE REVIEW

File: angular/Kanban/Swimlanes.md [hunk 1]
```diff
@@ -1,13 +1,13 @@
 ---
 layout: post
-title:  Swimlanes
-description: Swimlanes
+title:  Swimlanes in Angular Kanaban Control | Syncfusion
```
Comment (gowthamanthavasiyappan) [inline angular/Kanban/Swimlanes.md:3]: Remove 1 space before Swimlanes word

**Review by gowthamanthavasiyappan — CHANGES_REQUESTED** [review]
Commented

File: angular/Kanban/Swimlanes.md (hunk 1 above)
Comment (Yvone-Atieno) [reply angular/Kanban/Swimlanes.md]: removed

File: angular/Kanban/Swimlanes.md [hunk 2]
```diff
@@ -1,13 +1,13 @@
 ---
 layout: post
-title:  Swimlanes
-description: Swimlanes
+title: Swimlanes in Angular Kanaban Control | Syncfusion
```
Comment (gowthamanthavasiyappan) [inline angular/Kanban/Swimlanes.md:3]: Control name is Kanban, change and commit

**Review by gowthamanthavasiyappan — CHANGES_REQUESTED** [review]
Commented

File: angular/Kanban/Swimlanes.md (hunk 2 above)
Comment (Yvone-Atieno) [reply angular/Kanban/Swimlanes.md]: modified

**Review by gowthamanthavasiyappan — APPROVED** [review]
Approved

File: angular/Tab/Appearance-and-Styling.md [hunk 3]
```diff
@@ -1,13 +1,13 @@
 ---
 layout: post
-title: Appearance-and-Styling | Syncfusion
-description: appearance and styling
+title: Appearance and Styling in Angular Tab Control | Syncfusion
+description: Learn here about appearance and styling in Syncfusion Essential Angular TreeGrid Control, its elements, and more.
```
Comment (gowthamanthavasiyappan) [inline angular/Tab/Appearance-and-Styling.md:4]: Update the correct control name here

File: angular/Tile/Overview.md [hunk 4]
```diff
@@ -1,13 +1,13 @@
 ---
 layout: post
-title: Overview
-description: overview
+title: Overview in Angular Tile Control | Syncfusion
```
Comment (gowthamanthavasiyappan) [inline angular/Tile/Overview.md:3]: Overview of Angular Tile Control | Syncfusion

File: angular/Tile/Overview.md [hunk 5]
```diff
@@ -1,13 +1,13 @@
...
+description: Learn here more about Overview in Syncfusion Essential Angular Tile Control, its elements, and more.
```
Comment (gowthamanthavasiyappan) [inline angular/Tile/Overview.md:4]: Change `Overview of` instead of Overview in

File: angular/Tile/Overview.md [hunk 6]
```diff
@@ -1,13 +1,13 @@
...
 platform: Angular
 control: tile
 documentation: ug
 ---
 
-# Overview
+# Overview in Angular Tile
```
Comment (gowthamanthavasiyappan) [inline angular/Tile/Overview.md:10]: Overview of Angular Tile - change like this

**Review by gowthamanthavasiyappan — CHANGES_REQUESTED** [review]
Commented

File: angular/Tile/Overview.md (hunk 6 above)
Comment (Yvone-Atieno) [reply angular/Tile/Overview.md:10]: changed

File: angular/Tile/Overview.md (hunk 5 above)
Comment (Yvone-Atieno) [reply angular/Tile/Overview.md:4]: modified

File: angular/Tile/Overview.md (hunk 4 above)
Comment (Yvone-Atieno) [reply angular/Tile/Overview.md:3]: changed

File: angular/Tab/Appearance-and-Styling.md (hunk 3 above)
Comment (Yvone-Atieno) [reply angular/Tab/Appearance-and-Styling.md:4]: changed

**Review by Aishwarya-Ganesan — APPROVED** [review]
Changes are fine.

**Review by Aishwarya-Ganesan — APPROVED** [review]
Changes are fine.

**Review by gowthamanthavasiyappan — APPROVED** [review]
Approved

**Review by Aishwarya-Ganesan — APPROVED** [review]
Changes are fine.

**Pull request merged**